diamm/helpers/object_pagination.py

```diff
@@ -0,0 +1,30 @@
+from collections import OrderedDict
+import re
+
+from rest_framework.response import Response
+from rest_framework.pagination import PageNumberPagination
+
+class Object_Pagination(PageNumberPagination):
```

Comment:
Why are you using a custom pagination model? Why not use the [built-in](http://www.django-rest-framework.org/api-guide/pagination/) DRF pagination functionality?